srcpkgs/mosh/template

```diff
@@ -1,19 +1,24 @@
 # Template file for 'mosh'
 pkgname=mosh
-version=1.3.2
-revision=24
+version=20220318
```

Comment:
maybe a version like `1.3.2+20220318` or adding the commit hash or something would be better? If mosh gets another release in the future, I think that will break version comparison, as `20220318` is larger than `1`.

Reply:
Sounds good!